### PR TITLE
Wrap AXA-input (Lit-element based) instead of paper-slider (Polymer based)

### PR DESCRIPTION
**Pull request by joheriks**
Part of vaadin/flow#8924.

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/component-starter-flow/237)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Paper Slider
 
-Vaadin 14 Java integration of @polymer/paper-slider
+Vaadin 14 Java integration of @material/mwc-slider
```
Comment (pleku) [inline README.md:3]:
```suggestion
Vaadin Java integration of @material/mwc-slider
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 When creating a starter project from vaadin.com/start, a "npm module name"
 should be specified to create a Java component for it. If "npm module name" is left empty, a Java
-component is created for Polymer paper-slider component.
+component is created for Lit mwc-slider component.
```
Comment (pleku) [inline README.md:7]:
```suggestion
component is created for Lit based input-text component.
```

File: src/main/java/org/vaadin/artur/axainputtext/AXAInputText.java [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+@Tag("axa-input-text")
+@JsModule("@axa-ch/input-text/lib/index.js")
+@NpmPackage(value = "@axa-ch/input-text", version = "4.3.11")
 /*
  If you wish to include your own JS modules in the add-on jar, add the module
  files to './src/main/resources/META-INF/resources/frontend' and insert an
  annotation @JsModule("./my-module.js") here.
 */
-public class PaperSlider extends Component {
+public class AXAInputText extends Component {
```
Comment (pleku) [inline src/main/java/org/vaadin/artur/axainputtext/AXAInputText.java:16]: Should this be `AxaInputText` instead ? I can't find the source, but I do recall that all caps abbreviations are not that convenient for instance to autocomplete, AXAIT is more complex than AIT.

**Review by pleku — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joheriks) [reply README.md]: DOn.e

File: src/main/java/org/vaadin/artur/axainputtext/AXAInputText.java (hunk 3 above)
Comment (joheriks) [reply src/main/java/org/vaadin/artur/axainputtext/AXAInputText.java]: Done.

**Review by pleku — APPROVED** [review]

**Review by pleku — COMMENTED** [review]
> Reviewed 5 of 7 files at r2, 3 of 3 files at r3.
> *[Reviewable](https://reviewable.io/reviews/vaadin/component-starter-flow/237)* status: all files reviewed, 3 unresolved discussions (waiting on @joheriks and @pleku)

**Review by pleku — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/vaadin/component-starter-flow/237)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**